AVD loading takes all files recursively

The loading of AVDs is taking all files from a directory recursively which leads to loading wrong file.

```
[ERROR] [main] 2020-05-18 11:53:25,464 AVDService - Error loading avd "/home/bmarsaud/boxedroid/avd"
[ERROR] [main] 2020-05-18 11:53:25,466 AVDService - Error loading avd "/home/bmarsaud/boxedroid/avd/test_avd_27/config.ini"
[ERROR] [main] 2020-05-18 11:53:25,466 AVDService - Error loading avd "/home/bmarsaud/boxedroid/avd/test_avd_27/userdata.img"
[ERROR] [main] 2020-05-18 11:53:25,467 AVDService - Error loading avd "/home/bmarsaud/boxedroid/avd/test_avd/config.ini"
[ERROR] [main] 2020-05-18 11:53:25,467 AVDService - Error loading avd "/home/bmarsaud/boxedroid/avd/test_avd/userdata.img"
```

--- a/src/main/java/fr/bmarsaud/boxedroid/service/AVDService.java
+++ b/src/main/java/fr/bmarsaud/boxedroid/service/AVDService.java
@@ -106,7 +106,7 @@ public class AVDService {
     private void loadAVDs() {
         avds.clear();
 
-        List<String> avdDirs = IOUtils.getFilesInDir(avdsPath);
+        List<String> avdDirs = IOUtils.getFilesInDir(avdsPath, false);
         for(String avdDir : avdDirs) {
             String dirName = new File(avdDir).getName();
 

--- a/src/main/java/fr/bmarsaud/boxedroid/util/IOUtils.java
+++ b/src/main/java/fr/bmarsaud/boxedroid/util/IOUtils.java
@@ -6,10 +6,12 @@ import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.nio.file.Files;
+import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public class IOUtils {
     /**
@@ -56,17 +58,30 @@ public class IOUtils {
 
     /**
      * Get all files in a directory
-     * @param path The directory to returns the files
+     * @param dirPath The directory to returns the files
+     * @param recursively True the files are found recursively, False if we want only the first level
      * @return The list of files found
      */
-    public static List<String> getFilesInDir(String path) {
+    public static List<String> getFilesInDir(String dirPath, boolean recursively) {
         List<String> files = new ArrayList<>();
+        Path rootPath = Paths.get(dirPath);
+
         try {
-            files = Files.walk(Paths.get(path)).map(x -> x.toString()).collect(Collectors.toList());
+            Stream<Path> fileStream;
+
+            if(recursively) {
+                fileStream = Files.walk(rootPath);
+            } else {
+                fileStream = Files.walk(rootPath, 1);
+            }
+
+            files = fileStream.map(path -> path.toAbsolutePath().toString())
+                    .collect(Collectors.toList());
         } catch (IOException e) {
             e.printStackTrace();
         }
 
+        files.removeIf(filePath -> filePath.equals(rootPath.toAbsolutePath().toString()));
         return files;
     }
 }

